Sortable.js

```diff
@@ -167,7 +167,10 @@
 							scrollOffsetX = vx ? vx * speed : 0;
 
 							if ('function' === typeof(scrollCustomFn)) {
-								return scrollCustomFn.call(_this, scrollOffsetX, scrollOffsetY, evt);
+                var scrollFnResult = scrollCustomFn.call(_this, scrollOffsetX, scrollOffsetY, evt, touchEvt, el);
+								if (!options.keepOriginalScroll) {
```

Comment:
Instead of an additional option, you need to add that if the function returned `false` (or maybe better `"continue"`), then do not `return`.

Reply:
Thanks, will do!

Reply:
I know the probability is not high but if someone who implemented the scrollCustomFn returns false the original scroll will be executed after I make the suggested change. This makes it a possible breaking update. I'll go with `"continue"` even though it feels odd to return a string.